### PR TITLE
Fix testing and upgrade package

### PR DESCRIPTION
**Pull request by twxia**

### CODE REVIEW

File: test/TestElection.js [hunk 1]
```diff
@@ -42,7 +43,7 @@ contract('Election', (accounts) => {
 
       const candidateList = await election.getCandidatesList();
       const candidate = candidateList.find(candidate => candidate === accounts[1]);
-      const [vote, name] = await election.candidateData.call(1, candidate);
+      const { 0: vote, 1: name } = await election.candidateData.call(1, candidate);
```
Comment (wmin0) [inline test/TestElection.js:46]: 新版回的不是 array 了嗎._. 單純好奇

Comment (twxia) [reply test/TestElection.js:46]: yeap~ explode! check this doc:
https://github.com/trufflesuite/truffle/blob/800a2ffc6c3d609b672d17c0f6f4095ecb50a97c/notes/release/v5.0.0/src/breaking/index.md
`Functions that return multiple values now return an object with both named and indexed keys.`

Comment (RJ-software-outsourcing) [reply test/TestElection.js:46]: !!!!!

**Review by RJ-software-outsourcing — APPROVED** [review]

File: test/TestElection.js (hunk 1 above)
Comment (CHAOWEICHIU) [reply test/TestElection.js:46]: !!!

**Pull request merged**